src/pr-info.ts

```diff
@@ -108,9 +108,9 @@ export interface PrInfo {
     readonly author: string;
 
     /**
-     * True if the author or an owner wants us to merge the PR
+     * The last date the author or an owner asked us to merge the PR
      */
-    readonly mergeIsRequested: boolean;
+    readonly lastMergeRequestDate?: Date;
```

Comment:
Makes sense to provide better information to compute-... and have it make the decisions.